src/traces/box/calc.js

```diff
@@ -0,0 +1,138 @@
+'use strict';
```

Comment:
paste the license header from another file here.

Better yet, maybe you add a no-header-found clause in https://github.com/plotly/plotly.js/blob/master/tasks/header.js#L50 to do it automatically.